homeassistant/components/modbus/sensor.py

```diff
@@ -142,11 +142,13 @@ async def async_setup_platform(
             _LOGGER.error("Error in sensor %s structure: %s", entry[CONF_NAME], err)
             continue
 
-        if entry[CONF_COUNT] * 2 != size:
+        bytecount = entry.get(CONF_COUNT, 1) * 2
```

Comment:
When is `CONF_COUNT` not present in the `entry` dict?

Reply:
Is just to not repeat in error message

Reply:
In the future we will need that

Reply:
Sorry, I'm not following. Why don't we keep the old code?
```py
bytecount = entry[CONF_COUNT] * 2
```

Reply:
@nikito7 please do not add things for a future that might or might not come.

I prefer the old code, as MartinH suggests.

Reply:
But saves space.

Without variable I need duplicate COUNT

Reply:
But bytes stays or not?

This is needed for bytes....

Reply:
Choice is yours

Reply:
to me using (remark without the .get)
```
bytecount = entry(CONF_COUNT) * 2
if bytecount != size:
```
or 
```
if entry(CONF_COUNT) * 2 != size
```
is a matter of style, and I can live with both.

But MartinHjelmare has a good point, why touch code that do not need to be touched. So not changing that part is best.

@nikito7 I have helped you many times, and we both know you are new to python, so let me give you a good advice follow the suggestions.

Reply:
My point is that we should not use `dict.get` if we know that the key is in the dict. If we know that the key is in the dict we should do `dict[key]`.

Reply:
( this was a jan's idea. I like old style too )